### ISSUE TITLE
"Camera request timeout" when it's dark

### ISSUE THREAD

**Issue by vale-max** [intx82/a9-v720#26]
Hi there,
I tried to connect  to my camera and was getting a
`[  ERROR] [HTTP] Camera request timeout`

then I noticed that the camera was in a quite dark room, so I switched on the light and it connected.
Tried several times, switch off -> camera request timeout, switch on -> video was ok.

Is this the camera's fault or somehow the process of the incoming data fails to recognize a dark stream?

(No, the camera wasn't connected to the light switch :D )

**Comment by intx82** [comment]
yes, looks like camera fail, at least there are no checking for camera background in code.

Or this could be 'moving detection' enabled in camera, I saw such option in sources of original application. 

What is the camera version?
